k8s/base/ceramic-one/manifests/postgres-sts.yaml

```diff
@@ -36,6 +36,8 @@ spec:
                 secretKeyRef:
                   key: username
                   name: ceramic-postgres-auth
+            - name: PGDATA
+              value: "/var/lib/postgresql/ceramic_data"
```

Comment:
Is this path already mounted to some storage?

Reply:
Yes, if you look at [lines 59-62](https://github.com/ceramicstudio/simpledeploy/pull/42/files#diff-363613db95e18c9ad4ab95326dd503aee21ed84aefd3e53c72630981e1a2b8b0R59-R62), the PVC was being mounted at this path, but Postgres was not being told to use it for its data.